Makefile

```diff
@@ -9,8 +12,18 @@ test/cover: test
 test/cover: GOTEST_ARGS=-coverprofile=coverage.txt -covermode=atomic
 .PHONY: test/cover
 
+fuzz: FUZZTIME ?= 10s # The duration to run fuzz testing, default to 10s if unset.
+fuzz: # List all fuzz tests across the repo, and run them one at a time with the configured fuzztime.
+	@go list ./... | while read -r package; do \
+		go test -list '^Fuzz' "$$package" | grep '^Fuzz' | while read -r func; do \
+			echo "Running $$package $$func for $(FUZZTIME)..."; \
+			GOGC=$(GOGC) go test "$$package" -fuzz="$$func" -fuzztime=$(FUZZTIME); \
```

Comment:
We probably want to filter out all other normal tests here `-run '^$'` to avoid running all tests repeatedly for every iteration.